src/main/groovy/edu/oregonstate/mist/api/BuildInfoManager.groovy

```diff
@@ -0,0 +1,33 @@
+package edu.oregonstate.mist.api
+
+import io.dropwizard.lifecycle.Managed
+
+/**
+ * Created by georgecrary on 7/26/16.
+ */
+class BuildInfoManager implements Managed {
+    Info getInfo() {
+        info
+    }
+    private Info info = new Info()
+
+    @Override
+    public void start() throws Exception {
+        def stream = InfoResource.class.getResourceAsStream('/build.properties')
```

Comment:
If I understand it correctly, this method is meant to populate info when the application starts. Then why do you call getResourceAsStream from InfoResource class? would it make any difference if you call it from Resource or itself if possible?